packages/gamut/src/ContentContainer/index.tsx

```diff
@@ -0,0 +1,35 @@
+import cx from 'classnames';
+import React, { HTMLAttributes, ReactHTML } from 'react';
+
+import s from './styles.module.scss';
+
+export type ContentContainerProps = HTMLAttributes<HTMLElement> & {
```

Comment:
You know, I actually think we could (and therefore should) get rid of this:

```suggestion
export type ContentContainerProps = {
  className?: string;
```

Doing a search for `<ContentContainer(\n|\s+)` in the monolith, the only uses of things other than `className` and `el` are:

* `onClick`: only used in a deprecated Butterbar, and in an inaccessible way
* `data-testid`: only added _(by me, lol)_ for unit tests that could test on content instead
* `role="main"`: it'd be preferable to use `el="main"`

Thoughts?

Reply:
Ahh yeah I see what you're talking about. Committing your suggestion. Thanks!